integration_tests/follow_me.cpp

```diff
@@ -0,0 +1,263 @@
+#include <iostream>
+#include <cmath>
+#include <thread>
+#include <chrono>
+#include "integration_test_helper.h"
+#include "global_include.h"
+#include "dronecore.h"
+
+using namespace dronecore;
+using namespace std::chrono;
+using namespace std::this_thread;
+using namespace std::placeholders;
+
+
+void print(const FollowMe::Config &config);
+void send_location_updates(FollowMe &follow_me_handle, int count, float rate);
+
+TEST_F(SitlTest, FollowMe)
+{
+    DroneCore dc;
+
+    DroneCore::ConnectionResult ret = dc.add_udp_connection();
+    ASSERT_EQ(DroneCore::ConnectionResult::SUCCESS, ret);
+
+    // Wait for device to connect via heartbeat.
+    sleep_for(seconds(2));
+    Device &device = dc.device();
+
+    while (!device.telemetry().health_all_ok()) {
+        std::cout << "waiting for device to be ready" << std::endl;
+        sleep_for(seconds(1));
+    }
+
+    Action::Result action_ret = device.action().arm();
+    ASSERT_EQ(Action::Result::SUCCESS, action_ret);
+
+    device.telemetry().flight_mode_async(
+    std::bind([&](Telemetry::FlightMode flight_mode) {
+        FollowMe::TargetLocation last_location;
+        device.follow_me().get_last_location(last_location);
+
+        std::cout << "FlightMode: " << Telemetry::flight_mode_str(flight_mode)
```

Comment:
I think for integration tests we could re-use `LogInfo()`, etc. but we can do that in a later pull request because it's currently not consistent.